src/data/roadmapData.json

```diff
@@ -613,6 +610,81 @@
               "urlLabel": "Appium"
             }
           ]
+        },
+        {
+          "name": "NG Mobile QA Academy - Appium setup",
+          "data": [
+            {
+              "description": "The tutorial explains how to install Appium and how to setup project in Appium. You will need to ask for permission to access this Netguru internal material.",
+              "url": "https://drive.google.com/file/d/1O9MTIFZeTDxJpWsa7j-GAtpO0ROBGcky/view",
+              "urlLabel": "Tutorial movie"
+            },
+            {
+              "description": "",
+              "url": "https://drive.google.com/file/d/1KPwa9PXl5KLQVQsNSHn-2rq3lZ-07vwA/view?usp=drive_web",
+              "urlLabel": "Tutorial presentation"
+            }
+          ]
+        },
+        {
+          "name": "NG Mobile QA Academy - First tests in Appium",
+          "data": [
+            {
+              "description": "The tutorial explains how to write your first tests in Appium. You will need to ask for permission to access this Netguru internal material.",
+              "url": "https://drive.google.com/file/d/1tWmmnu5vh_saqfttr20_EaEHP7BtKrzY/view",
+              "urlLabel": "Tutorial movie"
+            },
+            {
+              "description": "",
+              "url": "https://drive.google.com/file/d/1w9J6VvoV9Y9cCE9oNbfU1j_-SFUhO6AD/view?usp=drive_web",
+              "urlLabel": "Tutorial presentation"
+            }
+          ]
+        },
+        {
+          "name": "NG Mobile QA Academy - Page Object Pattren in Appium",
```

Comment:
small typo - "Pattern" instead of "Pattren"

Reply:
fixed